revolv/templates/base/partials/category_preferences.html

```diff
@@ -0,0 +1,17 @@
+{% load staticfiles %}
+
+<div class="large-1 columns gray-bar-container">
+    <div class="gray-bar"></div>
+</div>
+<div class="large-11 columns choice-container">
+    <div class="choice-header">What type of projects should we invest your repayments in next?</div>
+    {% for category in categories %}
+        <div class="category-option-container {% if category in preferred_categories %}active{% endif %}" id="{{ category.id }}">
```

Comment:
Not a super good idea to put the category.id in the id of the element, since if we did this for something else on the page also, then there would be two elements on the same page with the same id, which is not supposed to happen and can sometimes create problems for javascript (esp when trying to find elements by id). Better idea would be to do `data-category-id={{category.id}}`: you can this access it with jQuery by doing `$(...).data("category-id")`; Read more at [http://stackoverflow.com/questions/7261619/jquery-data-vs-attr](http://stackoverflow.com/questions/7261619/jquery-data-vs-attr)